### ISSUE TITLE
Link to "linkcoursesections" can be invalid

### ISSUE THREAD

**Issue by t-schroeder** [moodleuulm/moodle-local_boostnavigation#34]
Hello,

I just noticed a very minor issue:

The setting "local_boostnavigation | insertcoursesectionscoursenode" contains a link to "/admin/search.php?query=linkcoursesections" which doesn't work if your Moodle instance is not at the root of the web server. Moodle's `$CFG->wwwroot` should be taken into account here.

Regards,
Tim

**Comment by abias** [comment]
Hi Tim,

thanks for raising this issue. Even very minor issues should be fixed.

I have to admit that it's not the first time that I produced this flaw (see https://github.com/moodleuulm/moodle-local_profiletheme/issues/6), I should know better :/

This is now fixed on https://github.com/moodleuulm/moodle-local_boostnavigation/commit/060ccb78987f2a098d0b3615462668eafb2e9896 and will be pushed to the Moodle plugins repo for Moodle 3.6 and updated in the german language pack in AMOS shortly.

Cheers,
Alex
